src/schema/comments.ts

```diff
@@ -832,7 +832,7 @@ export const resolvers: IResolvers<any, Context> = {
         });
       } catch (err) {
         if (err?.code !== TypeOrmError.DUPLICATE_ENTRY) {
-          ctx.log.error({ err }, 'failed to save report to database');
```

Comment:
I think we still need the logs, don't think the error does that but might be wrong

Reply:
It should be caught by `fastify.setErrorHandler` and respond with `500` https://github.com/dailydotdev/daily-api/blob/7c6d96af88937563dfadc28aaae33c21dd103651/src/index.ts#L43-L46

Reply:
Yap, Ante's right